keyboards/skippys_custom_pcs/rooboard65/config.h

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2021 <me@fjlaboratories.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x36b6 // Skippys Custom PCs
```

Comment:
```suggestion
#define VENDOR_ID    0x36B6 // Skippys Custom PCs
```

Also, this vender ID is very close to the currently issues VIDs, and makes me uncomfortable.

Reply:
> this vender ID is very close to the currently issues VIDs, and makes me uncomfortable.
This is merely what my client requested that I use. I have no say in the matter. If it were me, I would've put it under my own VID. Alas.